### ISSUE TITLE
Create my own data content in a format similar to the Visdata folder, enabling easy visualization different scenarios?

### ISSUE THREAD

**Issue by yoyo115956** [ueqri/vis4mesh#8]
Is there any way that I can generate my own data content in a format similar to the visdata folder, so that I can easily use it to visualise the scenarios that I need? I am really appreciate the function of vis4mesh.

**Comment by ueqri** [comment]
Yes. You can generate your own visualization data following the same format that Akita simulator uses. For more information, please refer to the vis4mesh tracer implementation in the [Akita repo](https://github.com/sarchlab/akita). Specifically, these two source files: (1) [meshnetworktracer.go](https://github.com/sarchlab/akita/blob/v3/noc/networking/noctracing/meshnetworktracer.go) and (2) [meshnetworkinfo.go](https://github.com/sarchlab/akita/blob/v3/noc/networking/noctracing/meshnetworkinfo.go).

**Comment by yoyo115956** [comment]
Thank you so much! Do you remember how you used the two GO's file internal functions here, because after researching for a day, I found that neither of the GO's file internal implementation methods were called by the code in Akita.

**Comment by ueqri** [comment]
It is not called by Akita itself. Akita is a simulation infrastructure repository, providing the components necessary for building a simulator. The end user who develops an Akita-based simulator can call the NoC tracer (or vis4mesh tracer). An example for reference is [Akkalat](https://github.com/ueqri/akkalat). It calls the NoC tracer [here](https://github.com/ueqri/akkalat/blob/0dbc0f746d3e579e75db280eae4b9ccb1a1e289a/runner/timingplatform.go#L398-L409).
